docs/content/components/collection/selectlist/selectlist.mdx

```diff
@@ -61,9 +139,26 @@ The `<SelectList>` lets you disable certain items, which is helpful if you don
 
 <PropsTable component="SelectListItem" />
 
-## Alternative components
+### SelectList.Label
+
+The label provides the primary text for each list item.
```

Comment:
for these sections also PropsTable would be needed, you don't need to write a sentence to it :)